core/text/table/table.odin

```diff
@@ -37,10 +43,19 @@ Table :: struct {
 	dirty: bool, // True if build() needs to be called before rendering
 
 	// The following are computed on build()
-	colw: [dynamic]int, // Width of each column (including padding, excluding borders)
+	colw: [dynamic]int, // Width of each column (excluding padding and borders)
 	tblw: int,          // Width of entire table (including padding, excluding borders)
 }
 
+ascii_width_proc :: proc(str: string) -> int {
+	return len(str)
+}
```

Comment:
Is this really needed, considering ascii is a subset of utf-8?

Reply:
Yes. `grapheme_count` has to iterate by UTF-8 codepoints then address several lookup tables by binary search. It's not fast at all. If someone knows in advance that all the text they're handling is ASCII, then it's better to use this proc rather than the Unicode version.